opengrok-web/src/main/webapp/repos.jspf

```diff
@@ -287,12 +302,26 @@ Copyright (c) 2015, 2018, Oracle and/or its affiliates. All rights reserved.
                                 if (cnt > 0 && ri.getParent() == null)
                                     // discard repositories without a parent url
                                     continue;
-                                if (cnt != 0) {
+                                boolean subrepository = !ri.getDirectoryNameRelative().equals(proj.getPath());
```

Comment:
looks like code duplication ?

Reply:
the whole thing with printing the repository is a code duplication (in groups and in without groups section) so this generally needs some better templating

I decided not to force it now and just fix the problem here.

Reply:
Fair enough. Filing a new issue would be in order.

Reply:
https://github.com/oracle/opengrok/issues/2685